.github/workflows/release_kolibri.yml

```diff
@@ -184,13 +184,13 @@ jobs:
     - name: Upload files to Google Cloud Storage
       uses: 'google-github-actions/upload-cloud-storage@v2'
       with:
-        path: 'dist/${{ matrix.filename }}'
+        path: 'dist/${{ matrix.filename }}/${{ matrix.filename }}'
```

Comment:
why is `${{ matrix.filename }}` in here twice?

Reply:
It's because of this fun quirk of the download-artifacts action: https://github.com/actions/download-artifact?tab=readme-ov-file#download-all-artifacts

Basically, artifacts get downloaded into a folder with the same name as the file, for reasons that I have still not adequately understood :)